packages/manager/modules/ng-layout-helpers/src/list/list-layout.utils.js

```diff
@@ -0,0 +1,355 @@
+import get from 'lodash/get';
```

Comment:
i've seen before few lodash module import in one line with lodash-es, is this better to do like here or with 1 import ?

Reply:
Yes `lodash-es` is supposed to be better in terms of performance but refactoring to use it everywhere should be done in a separate PR as it is a separate concern